.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length = 120
-extend-ignore = E203, W503, E501, E711, E712
+extend-ignore = E203, W503, E501, E711, E712, N803
```

Comment:
Similar to my comment in svix-libs it doesn't seem we break any of these rules so curious why you've added another exclude.